### ISSUE TITLE
need a new preprocessor macro to identify this fork

### ISSUE THREAD

**Issue by sezero** [open-watcom/open-watcom-v2#375]
This fork defines `__WATCOMC__` as 1300 at present.  On the other hand,
the builds from the official p4 repo (https://efbe.musca.uberspace.de/)
define `__WATCOMC__` as 2000.

We certainly need a new preprocessor macro to identify this very fork,
something like `__WATCOMC_2X__` or whatever.

**Comment by jmalak** [comment]
I don't think that it has sense to complicate our life by this numbering (because it is dead code).
correct numbering is 1290 for OW 1,9 and 1300 is for OW 2.0.
I don't thing that we will have any problem because official Open Watcom development is dead.
They are not capable to do upgrade(fixes) for OW 1.9 for years.
If official Open Watcom release new version we can add similar macro, but I think it will not happen.

**Comment by sezero** [comment]
OK then.  So I will go with something like
````
#if (__WATCOMC__ >= 1300 && __WATCOMC__ < 2000)
/* it is v2 fork at github */
````
... for the time being.
